src/Catty/Setup/Firebase/AppDelegateFirebaseExtension.swift

```diff
@@ -129,6 +130,18 @@ extension AppDelegate {
         crashlytics.record(error: error)
     }
 
+    @objc func projectSearchFailure(notification: Notification) {
+        let projectName = (notification.object as? ProjectLoadingInfo)?.visibleName ?? type(of: self).logNoValue
```

Comment:
This cast will never work. In the notification the `object` is of type `StoreProjectDownloader`, not `ProjectLoadingInfo` as assuming here.